PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/AdUnit.java

```diff
@@ -406,6 +406,9 @@ public void setPbAdSlot(String pbAdSlot) {
         configuration.setPbAdSlot(pbAdSlot);
     }
 
+    public void setGpid(@Nullable String gpid) {
```

Comment:
@ValentinPostindustria  Let's add a getter here as well. I see that there is no bid sense here, but just to be consistent with other properties.